.gitignore

```diff
@@ -1,9 +1,10 @@
 .coverage
 .mypy_cache
 .tox/
-.venv
+.venv/
```

Comment:
Why?

Reply:
Let me go through and clean some of these up. It looks like VSCode got a little change happy...